packages/patternfly-4/react-core/src/components/AboutModal/__snapshots__/AboutModalBoxCloseButton.test.tsx.snap

```diff
@@ -11,7 +11,6 @@ exports[`AboutModalBoxCloseButton Test 1`] = `
   >
     <TimesIcon
       color="currentColor"
-      noVerticalAlign={false}
```

Comment:
I'm not sure why these unrelated snapshots are updating, @jschuler @redallen do you know what the deal is here?

Reply:
react-icons was recently updated to include this prop, pull down latest from master and rerun yarn install